features/step_definitions/admin_node/packages_steps.rb

```diff
@@ -9,6 +9,10 @@
 end
 
 Given(/^all dependencies of installed packages are satisfied$/) do
-  admin_node.exec!("zypper", "verify --no-recommends")
+  admin_node.exec!(
+    "zypper",
+    "verify --no-recommends",
```

Comment:
-n verify to avoid that it asks a question (if e.g. a repo is unavailable or outdated..)